lib/runtime.ts

```diff
@@ -22,7 +22,7 @@ export default class Runtime {
   private tasksOnceComplete: Array<() => any> = [];
   // used for tracking computed dependencies
   public trackState: boolean = false;
-  public foundState: Set<State> = new Set();
+  public foundStates: Set<State> = new Set();
```

Comment:
"State" shouldn't be be plural in my opinion, much like "Sheep" the word is a "plurale tantum", where it is spelled the same plural as it is singular.

Reply:
For me it was always kinda wired because I thought there will be stored only one state
(And the plural of state is states https://www.wordhippo.com/what-is/the-plural-of/state.html)